third_party/python/BUILD

```diff
@@ -198,11 +198,6 @@ pip_library(
     version = "1.3.0",
 )
 
-python_wheel(
```

Comment:
This target fails to build for some reason, but think it was only needed for the wheel resolver tool which we don't need for v1 of the plugin.